### DISCUSSION TITLE
how can i fix it?

### DISCUSSION THREAD

**Discussion by lanyucomputer** [immortalwrt/immortalwrt#695] [Q&A] [answered]
Collected errors:
 * opkg_download: Failed to download https://mirrors.vsean.net/openwrt/releases/21.02.0/targets/ramips/mt7621/packages/Packages.gz, wget returned 8.
 * opkg_download: Failed to download https://mirrors.vsean.net/openwrt/releases/21.02.0/packages/mipsel_24kc/base/Packages.gz, wget returned 8.
 * opkg_download: Failed to download https://mirrors.vsean.net/openwrt/releases/21.02.0/packages/mipsel_24kc/luci/Packages.gz, wget returned 8.
 * opkg_download: Failed to download https://mirrors.vsean.net/openwrt/releases/21.02.0/packages/mipsel_24kc/packages/Packages.gz, wget returned 8.
 * opkg_download: Failed to download https://mirrors.vsean.net/openwrt/releases/21.02.0/packages/mipsel_24kc/routing/Packages.gz, wget returned 8.
 * opkg_download: Failed to download https://mirrors.vsean.net/openwrt/releases/21.02.0/packages/mipsel_24kc/telephony/Packages.gz, wget returned 8.

**Comment by lanyucomputer** [comment]
opkg update

**Comment by lanyucomputer** [comment]
the first time i can connected it ,today i can not, what is the matter with it  modle :  hc5962

**Comment by 1715173329** [comment]
镜像站规则设置有误导致无法正确获取文件。
如果您需要尽快恢复使用，请运行此命令并重新刷新列表。
`rm /overlay/upper/etc/opkg/distfeeds.conf`

**Reply by lanyucomputer** [reply]
> 镜像站规则设置有误导致无法正确获取文件。 如果您需要尽快恢复使用，请运行此命令并重新刷新列表。 `rm /overlay/upper/etc/opkg/distfeeds.conf`
连接主机...
连接主机成功


BusyBox v1.33.2 (2022-04-21 08:43:31 UTC) built-in shell (ash)

┌─────────────────────────────────────────────┐
│                                             │
│ mmmmm                         m       ""#   │
│   #   mmmmm mmmmm  mmm  mmm mm#mm  mmm  #   │
│   #   # # # # # # #" "# #" "  #   "   # #   │
│   #   # # # # # # #   # #     #   m"""# #   │
│ mm#mm # # # # # # "#m#" #     "mm "mm"# "mm │
│                                             │
│─────────────────────────────────────────────│
│              ImmortalWrt 21.02              │
└─────────────────────────────────────────────┘
root@ImmortalWrt:~# ls
root@ImmortalWrt:~# ping www.baidu.com
PING www.baidu.com (14.215.177.39): 56 data bytes
64 bytes from 14.215.177.39: seq=0 ttl=56 time=26.541 ms
64 bytes from 14.215.177.39: seq=1 ttl=56 time=26.253 ms
^C
--- www.baidu.com ping statistics ---
2 packets transmitted, 2 packets received, 0% packet loss
round-trip min/avg/max = 26.253/26.397/26.541 ms
root@ImmortalWrt:~# rm /overlay/upper/etc/opkg/distfeeds.conf
root@ImmortalWrt:~# 
root@ImmortalWrt:~# opkg update
Downloading https://mirrors.vsean.net/openwrt/releases/21.02.0/targets/ramips/mt7621/packages/Packages.gz
*** Failed to download the package list from https://mirrors.vsean.net/openwrt/releases/21.02.0/targets/ramips/mt7621/packages/Packages.gz

Downloading https://mirrors.vsean.net/openwrt/releases/21.02.0/packages/mipsel_24kc/base/Packages.gz
*** Failed to download the package list from https://mirrors.vsean.net/openwrt/releases/21.02.0/packages/mipsel_24kc/base/Packages.gz

Downloading https://mirrors.vsean.net/openwrt/releases/21.02.0/packages/mipsel_24kc/luci/Packages.gz
*** Failed to download the package list from https://mirrors.vsean.net/openwrt/releases/21.02.0/packages/mipsel_24kc/luci/Packages.gz

Downloading https://mirrors.vsean.net/openwrt/releases/21.02.0/packages/mipsel_24kc/packages/Packages.gz
*** Failed to download the package list from https://mirrors.vsean.net/openwrt/releases/21.02.0/packages/mipsel_24kc/packages/Packages.gz

Downloading https://mirrors.vsean.net/openwrt/releases/21.02.0/packages/mipsel_24kc/routing/Packages.gz
*** Failed to download the package list from https://mirrors.vsean.net/openwrt/releases/21.02.0/packages/mipsel_24kc/routing/Packages.gz

Downloading https://mirrors.vsean.net/openwrt/releases/21.02.0/packages/mipsel_24kc/telephony/Packages.gz
*** Failed to download the package list from https://mirrors.vsean.net/openwrt/releases/21.02.0/packages/mipsel_24kc/telephony/Packages.gz

Collected errors:
 * opkg_download: Failed to download https://mirrors.vsean.net/openwrt/releases/21.02.0/targets/ramips/mt7621/packages/Packages.gz, wget returned 8.
 * opkg_download: Failed to download https://mirrors.vsean.net/openwrt/releases/21.02.0/packages/mipsel_24kc/base/Packages.gz, wget returned 8.
 * opkg_download: Failed to download https://mirrors.vsean.net/openwrt/releases/21.02.0/packages/mipsel_24kc/luci/Packages.gz, wget returned 8.
 * opkg_download: Failed to download https://mirrors.vsean.net/openwrt/releases/21.02.0/packages/mipsel_24kc/packages/Packages.gz, wget returned 8.
 * opkg_download: Failed to download https://mirrors.vsean.net/openwrt/releases/21.02.0/packages/mipsel_24kc/routing/Packages.gz, wget returned 8.
 * opkg_download: Failed to download https://mirrors.vsean.net/openwrt/releases/21.02.0/packages/mipsel_24kc/telephony/Packages.gz, wget returned 8.
root@ImmortalWrt:~# 



i do it as you say ,but it does not working too

**Reply by 1715173329** [reply]
> i do it as you say ,but it does not working too

I got no idea why your feeds still pointed to mirror site, however it's back online now.

**Comment by lanyucomputer** [comment]
[https://mirrors.vsean.net](https://mirrors.vsean.net/openwrt/releases/21.02.0/packages/mipsel_24kc/telephony/Packages.gz)    website can not be open  is it matters?
 
 Access to mirrors.vsean.net was denied
You don't have authorization to view this page.
HTTP ERROR 403

**Reply by 1715173329** [reply]
> [https://mirrors.vsean.net](https://mirrors.vsean.net/openwrt/releases/21.02.0/packages/mipsel_24kc/telephony/Packages.gz) website can not be open is it matters?

Yes you can't open it with common browsers. To fight against robots, only `curl`, `uclient-fetch` and `wget` are allowed.

**Comment by lanyucomputer** [comment, marked as answer]
it is working now,thank you very much ,see you
